src/reducers/application.js

```diff
@@ -15,14 +17,15 @@ const handlers = {
   }),
 }
 
-// Actions
+// Actions // FIXME why isn't this in actions/*
```

Comment:
sometimes when actions are very dumb/simple, I feel it's not necessary to create dedicated action file (inspired by [redux duck pattern](https://medium.com/@scbarrus/the-ducks-file-structure-for-redux-d63c41b7035c) etc.)